plugins/woocommerce-blocks/assets/js/blocks/coming-soon/entire-site.scss

```diff
@@ -157,6 +97,14 @@ body:has(.woocommerce-coming-soon-banner) {
 	}
 }
 
+.woocommerce-coming-soon-entire-site,
+.woocommerce-coming-soon-image-gallery,
+.woocommerce-coming-soon-minimal-left-image {
+	.wp-block-loginout a {
+		text-decoration: none;
```

Comment:
I couldn't find a way to set text-decoration to none for the login button using site editor block settings so I had to add it to the CSS. I think we can raise a ticket in GB to improve this.